### PR TITLE
OCPBUGS-34175#adding new region

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions:
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-34175

Link to docs preview:
https://77079--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account.html#installation-azure-regions_installing-azure-account

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
Please add labels for 4.12+ as I don't have permissions, thanks.

**Comment by smunje1** [comment]
@brendan-daly-red-hat Peer review LGTM. As I am new to the peer review squad, I request that you wait until @bscott-rh gives approval.

Also, I am unable to see SME/QE approval in this PR or in Jira.  Could we add a Note in this PR description to clarify the same? Thank you!

**Comment by jinyunma** [comment]
LGTM

File: modules/installation-azure-regions.adoc
Comment (bscott-rh) [inline modules/installation-azure-regions.adoc:37]:
```suggestion
* `mexicocentral` (Mexico Central)
```
Looks like there is an extra space here :)

Comment (brendan-daly-red-hat) [reply modules/installation-azure-regions.adoc]: I removed the extra space - thanks.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**
